extensions/wikia/Search/classes/Services/ESFandomSearchService.php

```diff
@@ -17,6 +17,7 @@ class ESFandomSearchService extends AbstractSearchService {
 	const STORIES_IMAGE_URL_KEY = 'image';
 	const STORIES_URL_KEY = 'url';
 	const STORIES_COUNT_MAX = 6;
+	const STORIES_SEARCH_TEMPLATE = 'fandom_stories';
```

Comment:
Suggestion: Can we make this a WikiFactory variable?  It would make it easier to change this next time.